src/i18n/locales/en.json

```diff
@@ -730,7 +730,7 @@
     "guest_description": "Can only access projects that you specify choose. Tipically use for clients and stackholders.",
     "select_option": "Select an option",
     "subtitle": "Work together on projects",
-    "title": "Invite people to your workspace"
+    "title": "Invite people to your team"
```

Comment:
missing `invite.user_invited`

Reply:
Yeah I should probably run a `yarn migrations` in order to update all missing ones

Reply:
Done: bff2da46476970aeac8c2c03d0a6c5f8bcd00629